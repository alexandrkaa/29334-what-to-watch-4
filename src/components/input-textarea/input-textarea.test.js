import React from 'react';
import renderer from 'react-test-renderer';
import InputTextarea from './input-textarea.jsx';

describe(`<InputTextarea /> should match snapshot`, () => {
  const onTextAreaChange = jest.fn();
  const comment = `Some text`;
  it(`<InputTextarea /> should match snapshot (enabled)`, () => {
    const postCommentInProgress = false;
    const tree = renderer
      .create(
          <InputTextarea
            onTextAreaChange={onTextAreaChange}
            isDisabled={postCommentInProgress}
            comment={comment}
          />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it(`<InputTextarea /> should match snapshot (disabled)`, () => {
    const postCommentInProgress = true;
    const tree = renderer
      .create(
          <InputTextarea
            onTextAreaChange={onTextAreaChange}
            isDisabled={postCommentInProgress}
            comment={comment}
          />
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
