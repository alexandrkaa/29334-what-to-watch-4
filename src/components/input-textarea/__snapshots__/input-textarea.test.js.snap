// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InputTextarea /> should match snapshot <InputTextarea /> should match snapshot (disabled) 1`] = `
<textarea
  className="add-review__textarea"
  disabled={true}
  id="review-text"
  name="review-text"
  onChange={[Function]}
  placeholder="Review text"
  value="Some text"
/>
`;

exports[`<InputTextarea /> should match snapshot <InputTextarea /> should match snapshot (enabled) 1`] = `
<textarea
  className="add-review__textarea"
  disabled={false}
  id="review-text"
  name="review-text"
  onChange={[Function]}
  placeholder="Review text"
  value="Some text"
/>
`;
